cppcache/src/PdxType.cpp

```diff
@@ -556,12 +556,23 @@ bool PdxType::Equals(std::shared_ptr<PdxType> otherObj) {
   return true;
 }
 
-bool PdxType::operator<(const PdxType& other) const {
-  auto typeIdLessThan = this->m_geodeTypeId < other.m_geodeTypeId;
-  auto typeIdsBothZero =
-      (this->m_geodeTypeId == 0) && (other.m_geodeTypeId == 0);
-  auto classnameLessThan = this->m_className < other.m_className;
-  return (typeIdLessThan || (typeIdsBothZero && classnameLessThan));
+bool PdxType::operator==(const PdxType& other) const {
```

Comment:
How does this differ from the `PdxType::Equals` method? Is there a purpose for the `PdxType::Equals` anymore?

Reply:
I think we can get rid of `PdxType::Equals`